backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -633,4 +697,10 @@ private void fillOrStrokePath (DrawType drawType) {
 
 		void downloadFailed (Throwable t);
 	}
+
+	public interface LoadImageAsyncListener {
```

Comment:
This probably doesn't need to be public, unless it is intended for external use. The only usage I can see is fully within code here.